cmd/zoekt-webserver/main.go

```diff
@@ -150,7 +152,10 @@ func main() {
 	}
 
 	initializeJaeger()
-	initializeGoogleCloudProfiler()
+	err := profiler.Init(ServiceName, zoekt.Version, -1)
+	if err != nil {
+		log.Printf("could not initialize profiler: %s", err.Error())
```

Comment:
if we only ever log the error, can we improve the ergonomics here and just log internally in profiler.Init and not return an error.